packages/preview/tlseinp-report/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "tlseinp-report"
+version = "0.1.0"
+entrypoint = "tlseinp.typ"
+authors = ["Killian Falguiere <@killian-fal>"]
+license = "Apache-2.0"
+categories = ["report"]
+repository = "https://github.com/Killian-fal/tlseinp-typst-template"
+keywords = ["inp", "toulouse-inp", "n7", "a7", "enseeiht", "ensat", "ensiacet", "engineering", "ingenieur", "rapport", "report", "tp", "td", "projet", "compte-rendu"]
+disciplines = ["engineering", "computer-science", "mathematics", "physics"]
+description = "Template for Toulouse INP students."
+
+[template]
+path = ""
```

Comment:
Templates should go in their own subdirectory. The content of this directory will be copied when creating a project from your template, so it should probably contain a new file that imports `@preview/tlseinp-report:0.1.0` and set ups the document with a minimal working example.